SwiftSyft/Classes/SyftWebSocket_iOS12.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public class SyftWebSocketIOS12: SyftWebSocket, WebSocketDelegate {
+    var socket: WebSocket!
```

Comment:
You can create this in the initialzer init(url: URL, pingInterval: Double) so it can be `let` instead of `var`.

Reply:
it's not clear for me, please check the last commit and clarify it again if it's not fixed. Thanks!